modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -399,6 +402,12 @@ public void testAddSystemObjectRelationship() throws Exception {
 				StringUtil.randomId(), false,
 				ObjectRelationshipConstants.TYPE_MANY_TO_MANY, null));
 
+		_testAddObjectRelationshipManyToMany(
+			ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+			_modifiableSystemObjectDefinition, _objectDefinition1, false);
+		_testAddObjectRelationshipManyToMany(
+			ObjectRelationshipConstants.DELETION_TYPE_PREVENT,
+			_modifiableSystemObjectDefinition, _objectDefinition1, true);
 		_testAddObjectRelationshipManyToMany(
 			ObjectRelationshipConstants.DELETION_TYPE_CASCADE,
 			_objectDefinition1, _systemObjectDefinition2, false);
```

Comment:
@p-albuquerque the `DELETION_TYPE_PREVENT` should be placed after the `DELETION_TYPE_DISASSOCIATE` to follow the existing pattern